package.json

```diff
@@ -18,15 +18,14 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "^1.45.0",
+    "@playwright/test": "^1.49.0",
     "@types/node": "^20.11.17",
-    "browserstack-node-sdk": "^1.34.18",
+    "browserstack-node-sdk": "^1.34.23",
     "dotenv": "^16.0.3",
     "eslint": "^8.44.0",
     "eslint-config-prettier": "^8.8.0",
     "husky": "^8.0.3",
     "minimist": "^1.2.8",
-    "playwright": "^1.45.0",
```

Comment:
QUESTION: No playwright??

Reply:
we just need @playwright/test for the playwright dependencies